frontend/src/features/admin-form/create/featureTour/FeatureTourTooltip.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { BoxProps } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { fullScreenDecorator } from '~utils/storybook'
+
+import { FEATURE_TOUR } from './constants'
+import { FeatureTourTooltip } from './FeatureTourTooltip'
+
+export default {
+  title: 'Pages/FeatureTour/Tooltip',
+  decorators: [fullScreenDecorator],
+  parameters: {
+    layout: 'fullscreen',
+    chromatic: { pauseAnimationAtEnd: true, delay: 50 },
+  },
+} as Meta
+
+const mockStep = {
+  content: `${FEATURE_TOUR[0].content}`,
+  title: `${FEATURE_TOUR[0].title}`,
+}
+
+const mockButtonProps = {
+  onClick: () => {
+    return
+  },
+}
+
+const Template: Story = () => {
+  return (
+    <FeatureTourTooltip
+      step={mockStep}
+      tooltipProps={{} as BoxProps}
+      primaryProps={mockButtonProps}
+      skipProps={mockButtonProps}
+      isLastStep={false}
+    />
+  )
+}
+
+export const BasicUsage = Template.bind({})
```

Comment:
Can probably add more stories, such as if it is the last step.

Can even mock with a useState inside the template story to move between steps, and pass in an onclick handler to advance steps :)

Reply:
Updated as per comment